src/Components/SmartComponents/CveDetail/CveDetailPageHeader.js

```diff
@@ -54,24 +54,24 @@ const CveDetailPageHeader = () => {
   const hasMetadata = description !== 'unknown';
 
   useEffect(() => {
-    dispatch(fetchCveDetails(match.params.cveId));
+    dispatch(fetchCveDetails(params.cveId));
   }, []);
 
   return (
     <PageHeader style={{ paddingBottom: 0 }}>
       <Breadcrumb className="pf-u-mb-md">
         <BreadcrumbItem>
-          <Link to="/">Vulnerability</Link>
+          <Link to="..m">Vulnerability</Link>
```

Comment:
`m` is probably a typo, or?

Reply:
Correct, thanks for the find :smile:

Reply:
The reason I didn't use the `<InsightsLink>` component is that the generated path is not correct, propably because it's on Openshift. I'll check it out a bit later